.github/workflows/buildService.yml

```diff
@@ -0,0 +1,64 @@
+name: Build Service
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths-ignore: ["*.md"]
+    branches: ["master", "next"]
+  push:
+    paths-ignore: ["*.md"]
+    branches: ["master", "next"]
+
+jobs:
+  BuildPackage:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Set up Rust Toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: stable
+      - name: Get StartOS Hash
+        run: echo "STARTOSHASH=$(git ls-remote https://github.com/Start9Labs/start-os --branch sdk | awk '{ print $1}')" >> $GITHUB_ENV
+      - name: Cache Packages
+        uses: buildjet/cache@v3
+        id: packageCache
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-vaultwarden-${{ env.STARTOSHASH }}
+      - name: Install Dependencies
+        run: |
+          sudo snap install yq deno
+          sudo apt-get install -y build-essential openssl libssl-dev libc6-dev clang libclang-dev ca-certificates
+      - name: Checkout services repository
+        uses: actions/checkout@v3
+      - name: Install Package Manager
+        id: packageManager
+        if: steps.packageCache.outputs.cache-hit != 'true'
```

Comment:
```
if: steps.packageCache.outputs.cache-hit != 'true'
```
Because of this, it makes this workable. But the long term solution is to pull this from something like a release on the start-os